### ISSUE TITLE
[a11y]: Contrast on tiles/cards is below 3:1

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#11351]
### Package

carbon-components

### Browser

Chrome

### Operating System

MacOS

### Package version

Take your pick :)

### React version

take your pick :)

### Automated testing tool and ruleset

n/a

### Assistive technology

n/a

### Description

As discussed on some recent cards, there are a variety of implementations of cards in IBM:
- Some are actionable
- some are sortable
- some are selectable
- some are just holding static text
- some are themselves non-actionable but contain a link or other actionable item
- some have multiple actionable items

Especially when we get into Carbon flavours, similarly functioning cards share very few attributes, in regard to placement, stroke width, stroke strength, spacing, etc., etc.
I think overall there are challenges for all users trying to discriminate between actionable cards/tiles and non-actionable content holders. But for users with disabilities, the problems of discernment and interaction become more pronounced.

One clear way to improve this situation is to undertake a clearer division between when a card is actionable, and when it is just a visual container for other content. There is a requirement called **Non-text Contrast** that I think can help with this.
### Visual information required to identify user interface components and states
[1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) requires that controls (AKA [user interface components](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#dfn-user-interface-component)) have a 3:1 contrast against adjacent colors for visual information **required to identify** them. So, if a card is actionable -- a control -- it should have something like a stroke around it that contrasts 3:1 against the page background so that users can understand it is actionable. There are other ways besides strokes to do this, for instance the entire card could be 3:1 against the background.

And of course it follows that although there is no requirement saying this, a tile/card that is NOT actionable shouldn't have a 3:1 contrast. That way everyone would be able to better tell when there is an intended interaction on the page, and when it is just 'decorative'.

This isn't a magic bullet to solve all the challenges making consistent and beautiful cards, but I recommend this basic rule of 3:1 contrast as a way of better defining intended interaction, while improving the accessibility of components for everyone :)

### WCAG 2.1 Violation

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

### CodeSandbox example

https://react.carbondesignsystem.com/?path=/story/components-tile--default

### Steps to reproduce

1. Look at just about any card/tile in Carbon, including https://react.carbondesignsystem.com/?path=/story/components-tile--default
2. Use a contrast checker to check the card for 3:1 contrast against adjacent colors.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
> So, if a card is actionable -- a control -- it should have something like a stroke around it that contrasts 3:1 against the page background so that users can understand it is actionable.

@mbgower just to confirm here, this stroke needs to be applied to the default state, independent of focus state? For example, clickable tile has no stroke by default and then on focus has a stroke that meets 3:1 contrast

| default | focus |
| - | - |
| ![image](https://user-images.githubusercontent.com/3360588/167033391-364efbdd-724c-472c-92d9-ac3c44a1ed73.png)
 | ![image](https://user-images.githubusercontent.com/3360588/167033422-2dc5a202-2edc-475e-bb0a-abf6a486f18c.png) |

**Comment by mbgower** [comment]
> this stroke needs to be applied to the default state, independent of focus state?

That's correct, the visual information that distinguishes the control has to be 3:1 in all states, including unfocused and unselected.
So in your two illustrations, the clickable tile in the default (unfocused) state either needs to have its grey fill bumped up to 3:1, or have a stroke added around the gray that is 3:1 against the page background, or some other contrasted indicator  that means to the user 'this is an actionable tile'
It's really useful to look at the definition of user interface component to help with this:
> a part of the content that is perceived by users as a single control for a distinct function

BTW, normally focus indicators are offset outlines, so they aren't just replacing the normal stroke, they're outside it. That said, for now you can get away with replacing a thin grey line contrasting 3:1 with a thicker blue line that also contrasts 3:1. However, there are bigger requirements for Focus appearance coming up with the release of WCAG 2.2, so I would recommend offsetting focus indicators. We can pick that up in discussions later; it's a separate consideration.

**Comment by ljcarot** [comment]
Let's move this in to next Sprint please.  Thy is best person to work on this.  Thanks @sstrubberg @tay1orjones

**Comment by thyhmdo** [comment]
Updating these actionable tiles with a stroke: `border-strong` (gray 50) 
- Clickable 
- Expandable (no interaction)
- Selectable (checkbox)
- Selectable (radio)

![Tile - White theme](https://user-images.githubusercontent.com/77697351/176736193-380b9196-68bb-4455-b408-506c4a1dbc71.png)

**Comment by dakahn** [comment]
@mbgower diving into the guidance for non-text contrast and based on the boundaries section--
> This success criterion does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed. 

it would seem to me that cards with stuff inside them indicating a control would pass. A blank card would need sufficient contrast against adjacent colors but this card
![image](https://user-images.githubusercontent.com/40970507/178610431-7944c50c-5cd2-45f7-9458-bb19b0700ee3.png)
and this button used as a passing example in the Success Criterion
![image](https://user-images.githubusercontent.com/40970507/178610471-2d65fd7b-66c1-494a-ae3a-430f89851e93.png)
are functionally identical, right?

**Comment by aagonzales** [comment]
Next steps: @thyhmdo 
- [ ] Define MVP to resolve issue (adding an icon indicator to all variants?) - Send to development 
- [ ] Explore other possible visual indicators to show tiles are operable (blue sky, anything is possible)
   - What can we do that doesn't include a 3:1 border around the entire click target?
   - How do we distinguish been a non-clickable container and an operable tile/card?

**Comment by mbgower** [comment]
Yes. However, the 'button' has a lot of other ways it is getting a bit of a boost
The very fact it is separated by white space from any other content immediately causes some level of cue that it is likely interactive.

You don't get that with tiles. Their content is frequently laid out in a grid, but so is content that is NOT actionable. So how do users differentiate?

**Comment by sstrubberg** [comment]
Connecting to #10493

**Comment by tay1orjones** [comment]
We're tracking the implementation of this over in https://github.com/carbon-design-system/carbon/issues/12278

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/issues/12278
